eppo/src/androidTest/java/cloud/eppo/android/EppoClientTest.java

```diff
@@ -781,7 +789,7 @@ private void waitForPopulatedCache() {
       while (!cachePopulated) {
         if (System.currentTimeMillis() > waitEnd) {
           throw new InterruptedException(
-              "Cache file never populated; assuming configuration error");
+              "Cache file never populated or smaller than expected 8000 bytes; assuming configuration error");
```

Comment:
wasted a bunch of time trying to figure this out (my test data branch had a very small config).

Reply:
where is this "expected 8000 bytes" coming from? first time I'm hearing about it I think

Reply:
There's a check below that checks for a minimum cache file size of 8k

Reply:
[line 794/995](https://github.com/Eppo-exp/android-sdk/pull/137/files/b1c6a6bcf14f2e8b77e58869296d2bf5bf192468#diff-38172de297d3530a02668c9bbc0f4ca214d4ed7386f4e7b73b8eae0f477245b0R794) cc: @felipecsl